### ISSUE TITLE
training loss decrease very slowly using score model type

### ISSUE THREAD

**Issue by smilebetterworld** [ZhengYinan-AIR/Diffusion-Planner#55]
@ZhengYinan-AIR , hi , when using score model type, the training loss is large and decrese very slowly, and the inference result is not correct when testing,  I want to know whether you have verified it works well using score model type?

**Comment by jessapinkman** [comment]
same, it doesn't work for me. I use 1M trainning samples and the loss cannot decrease and remain large 7~9.

**Comment by LiuisStupid** [comment]
> [@ZhengYinan-AIR](https://github.com/ZhengYinan-AIR) , hi , when using score model type, the training loss is large and decrese very slowly, and the inference result is not correct when testing, I want to know whether you have verified it works well using score model type?

hello, are you sure your training loss is "decreasing"? I found that the training loss just oscillates around "8.0", which is the squared sum of ego(x y sin cos) + mean_neighbor(x y sin cos), because they are normal-distributed, so the sum-mean keeps "8", the model is predicting nothing during the training process

**Comment by HerrSchnapps** [comment]
same 

<img width="1665" height="468" alt="Image" src="https://github.com/user-attachments/assets/77cb1305-d64c-4b84-b783-f258ccff2692" /> 

the loss can not convergence

**Comment by huzhengwei** [comment]
same problem

**Comment by Yongqi7** [comment]
@HerrSchnapps I have the same loss distribution. Have you resolved this issue?

**Comment by HerrSchnapps** [comment]
> [@HerrSchnapps](https://github.com/HerrSchnapps) I have the same loss distribution. Have you resolved this issue?

No, the loss can not be convergenced. And I can not repeat the performance either. In this project there are 2 modes and 2 groups of code, both of them seems can not match with the paper.
